[2024-03-18 22:36:24,602] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: opcion_2.extraer_datos2 scheduled__2024-03-17T22:36:15.609958+00:00 [queued]>
[2024-03-18 22:36:24,608] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: opcion_2.extraer_datos2 scheduled__2024-03-17T22:36:15.609958+00:00 [queued]>
[2024-03-18 22:36:24,609] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-03-18 22:36:24,610] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-03-18 22:36:24,610] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-03-18 22:36:24,623] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): extraer_datos2> on 2024-03-17 22:36:15.609958+00:00
[2024-03-18 22:36:24,626] {standard_task_runner.py:52} INFO - Started process 244 to run task
[2024-03-18 22:36:24,629] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'opcion_2', 'extraer_datos2', 'scheduled__2024-03-17T22:36:15.609958+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/tareas2.py', '--cfg-path', '/tmp/tmpkv8swmg_', '--error-file', '/tmp/tmp3h4fyv4u']
[2024-03-18 22:36:24,631] {standard_task_runner.py:80} INFO - Job 27: Subtask extraer_datos2
[2024-03-18 22:36:24,685] {task_command.py:371} INFO - Running <TaskInstance: opcion_2.extraer_datos2 scheduled__2024-03-17T22:36:15.609958+00:00 [running]> on host 1f82df9fcae4
[2024-03-18 22:36:24,754] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=opcion_2
AIRFLOW_CTX_TASK_ID=extraer_datos2
AIRFLOW_CTX_EXECUTION_DATE=2024-03-17T22:36:15.609958+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-17T22:36:15.609958+00:00
[2024-03-18 22:36:26,215] {python.py:173} INFO - Done. Returned value was: None
[2024-03-18 22:36:26,231] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=opcion_2, task_id=extraer_datos2, execution_date=20240317T223615, start_date=20240318T223624, end_date=20240318T223626
[2024-03-18 22:36:26,253] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-03-18 22:36:26,280] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
